routes/web.php

```diff
@@ -44,6 +44,10 @@
     Route::get('/bid/{id}', 'RealopsController@bid')->middleware('auth:realops')->middleware('toggle:realops_bidding');
     Route::get('/cancel-bid/{id}', 'RealopsController@cancelBid')->middleware('auth:realops');
 });
+
+Route::prefix('live')->middleware('toggle:live_event')->group(function () {
+    Route::get('/', 'FrontController@showLiveEventInfo');
+});
```

Comment:
I like that we're using feature toggles here, but we may want to discuss rethinking how we implement these. The goal of toggles should be to eventually be able to remove them completely - not use them as a crutch. I.e. they should be to release a feature and be able to turn it on/off based on whether or not it breaks or if we want to release a feature on a day other than a deploy day. I know we started this pattern with realops, but the purpose of that was to build a working solution fast - not really something we should follow. I'm fine with this for now, but eventually we need to think about refactoring these into a way where we can eventually remove the toggle from the code.

Reply:
I removed all uses of the feature toggles except for one (displaying menu item in the navbar). I couldn't find a way to make my publish flag globally-accessible to all of the FrontController methods... just a pain. Lmk if you see a better way to do this.

I just added a toggle on the admin page so the person editing the info can toggle it on/off.

It also made sense to remove toggle guards from the routes, as even if the event info is turned off, we don't want the route to break. Instead of the route breaking, a default message is now displayed for a more graceful let down.

Reply:
This looks good!

Reply:
Done - this eliminated use of the feature toggle all together. Of note, this required that I make the constructor protected instead of private so it can instantiate itself when called statically.